### PR TITLE
Connect Individual Show Page to API

### LINKED ISSUE #127: Set up Mock API to generate and pass show info for the Individual Show page
Related to User Story #129

### PR DESCRIPTION
**Pull request by sam-eng**
draft PR for now since I'm still working on this

resolves #127

### CODE REVIEW

**Comment by sam-eng** [comment]
Something to note: there's a bug where the current status and platform for a show won't load when the page loads. I spent... a while trying to get this fixed and couldn't get it to work, so I'm putting it into its own issue.

**Comment by almazhankapan** [comment]
Looks really good, thank you! Just one small edit for the font-size

File: front-end/src/IndividualShow.css [hunk 1]
```diff
@@ -212,4 +209,25 @@ span {
...
+  }
+
+  .platform-select,
+  .status-select {
+    max-width: 50%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 1em;
+  }
+}
```
Comment (almazhankapan) [inline front-end/src/IndividualShow.css:227]: i've written a comment about the change for the font-size, thank you!

**Review by almazhankapan — CHANGES_REQUESTED** [review]

**Review by almazhankapan — APPROVED** [review]

**Comment by almazhankapan** [comment]
great work!

**Pull request merged**